x-pack/plugins/apm/server/routes/errors/get_error_groups/get_error_group_sample_ids.ts

```diff
@@ -72,7 +79,10 @@ export async function getErrorGroupSampleIds({
     'get_error_group_sample_ids',
     params
   );
-  const errorSampleIds = resp.hits.hits.map((item) => item._source.error.id);
+  const errorSampleIds = resp.hits.hits.map((item) => {
+    const source = item._source as APMError;
```

Comment:
any specific reason to extract this in 2 steps ?

Reply:
For the typing, need to cast here also

Reply:
<img width="440" alt="image" src="https://github.com/elastic/kibana/assets/31922082/1237d43e-9328-48df-90df-3f7edaf176a4">